CHANGELOG.md

```diff
@@ -15,6 +15,15 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
+# v1.7.1 (Apr 29, 2025)
+## Release Summary
+This release addresses bug fixes to improve stability and user experience for Dell PowerScale.
+
+## Bug Fixes
+* [Stabilizing the user addition to the role resource.](https://github.com/dell/dell-terraform-providers/issues/24)
+* [Reading addition field from networkpool resource.](https://github.com/dell/dell-terraform-providers/issues/23)
+* [Handling user present in non-system zone while using namespaceacl resource.](https://github.com/dell/dell-terraform-providers/issues/18)
```

Comment:
We also need to mention in the enhancements that we've added an extra filter attribute named SID in the user datasource to filter the users based on SID